can/interfaces/virtual.py

```diff
@@ -88,13 +88,18 @@ def send(self, msg, timeout=None):
         msg_copy = deepcopy(msg)
         msg_copy.timestamp = time.time()
         msg_copy.channel = self.channel_id
+        msg_copy.is_rx = True
 
         # Add message to all listening on this channel
         all_sent = True
         for bus_queue in self.channel:
+            msg_to_send = msg_copy
+            if bus_queue is self.queue and self.receive_own_messages:
+                msg_to_send = deepcopy(msg_copy)
```

Comment:
A regular copy would suffice here.

Reply:
@christiansandberg Isn't deepcopy better than copy? I mean doing msg.data = xx after receiving a message in one queue will modify the message for all other virtual interfaces. 

Also the original implementation sends the same message instance on every virtual bus. Should we change that to send a different instance since the same argument applies there.